framework/wazuh/core/cluster/master.py

```diff
@@ -1020,7 +1030,41 @@ def __init__(self, **kwargs):
         self.integrity_already_executed = []
         self.dapi = dapi.APIRequestQueue(server=self)
         self.sendsync = dapi.SendSyncRequestQueue(server=self)
-        self.tasks.extend([self.dapi.run, self.sendsync.run, self.file_status_update, self.agent_groups_update])
+
+        # Initialize Indexer client if available
+        self.indexer_client = None
+        if AsyncOpenSearch:
+            try:
+                # Get configuration from environment variables (standard in Wazuh Docker env)
+                indexer_url = os.environ.get('INDEXER_URL', '')
+                indexer_user = os.environ.get('INDEXER_USERNAME', 'admin')
+                indexer_pass = os.environ.get('INDEXER_PASSWORD', 'admin')
```

Comment:
We need to discuss how to get the credentials because this is not the standard way